src/main/java/longah/Command.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * The Command class parses user input into a keyword and arguments.
+ */
+public class Command {
+    private String[] parts;
+
+    /**
+     * Constructs a new Command instance with the given input.
+     *
+     * @param input The user input.
+     */
+    public Command(String input) {
+        this.parts = input.split(" ");
+    }
+
+    /**
+     * Gets the command keyword.
+     *
+     * @return The command keyword.
+     */
+    public String getKeyword() {
```

Comment:
In the same vein as above, would be nice to have a separate class handling user input as opposed to this class handling both user input and command execution.

Reply:
For now can just implement the `abstract` class itself, specific followup classes for individual commands can be left to those performing the implementation of the specific tasks. That way, the latter class can be used for task identification as well so we reduce clutter in this class.